packages/templates/manifest.json

```diff
@@ -1,28 +1,36 @@
 {
     "templates": [
+        {
+            "name": "getting-started-ts",
+            "description": "Getting started example [TypeScript]"
+        },
+        {
+            "name": "getting-started-js",
+            "description": "Getting started example [JavaScript]"
+        },
         {
             "name": "cheerio-ts",
-            "description": "Crawlee cheerio template [TypeScript]"
+            "description": "CheerioCrawler template project [TypeScript]"
```

Comment:
Maybe lets drop the word `template` too?

Reply:
Dunno, to me it signifies that its some boilerplate for extending and not an example, but I don't have a strong opinion about that.